WordPress/Sniffs/VIP/PostsPerPageSniff.php

```diff
@@ -54,17 +63,19 @@ public function getGroups() {
 	 */
 	public function callback( $key, $val, $line, $group ) {
 		$key = strtolower( $key );
+		$this->posts_per_page = (int) $this->posts_per_page;
```

Comment:
Either align the `=` signs or add a blank line before this line.

Reply:
Will align the equal signs, other sniffs are aligning that way so it's better for the consistency.